### DISCUSSION TITLE
Trim results as you type

### DISCUSSION THREAD

**Discussion by Dindin-Rules** [microsoft-search/pnp-modern-search#3651] [Q&A]
Good day, 

I recently started using this add'in and setup a list with a PnP-Search Results and a PnP - Search Box tied together so the search box is trimmed by the search results.

The client asked if the data displayed can be trimmed by what you type, as you type it vs when you hit enter. 

Is that even a possibility? I didn't see it when I searched the forum questions but I may have missed it.

**Comment by wobba** [comment]
Today it's not available, and if you do this with a lot of users you will likely be throttled on the API's doing this as well.

**Reply by Dindin-Rules** [reply]
Ok. Thanks
